pkg/virt-handler/device-manager/generic_device.go

```diff
@@ -237,6 +247,7 @@ func (dpi *GenericDevicePlugin) ListAndWatch(e *pluginapi.Empty, s pluginapi.Dev
 	s.Send(&pluginapi.ListAndWatchResponse{Devices: dpi.devs})
 
 	for {
+		done := false
```

Comment:
Maybe we want to declare this outside of `for` scope.

Reply:
Done.